packages/core/src/services/ubi/community.ts

```diff
@@ -1236,7 +1237,49 @@ export default class CommunityService {
             type: QueryTypes.SELECT,
         });
 
-        const results: ICommunityPendingDetails[] = rawResult.map((c) => ({
+        const callData = rawResult.reduce((acc, el) => {
```

Comment:
Sorry, this is an old deprecated method.
only /list is used now, with `status=pending` to filter by pending.

Once you try to do this within /list, it will break the page count because it will return the wrong number of results. A workaround is needed

Reply:
could you validate again, please?